.github/workflows/ci.yml

```diff
@@ -300,4 +300,6 @@ jobs:
         with:
           name: slang-build-${{matrix.os}}-${{matrix.platform}}-${{matrix.compiler}}-${{matrix.config}}
           # The install directory used in the packaging step
-          path: build/dist-${{matrix.config}}/**/ZIP/slang/*
+          path: |
+            build/dist-${{matrix.config}}/**/ZIP/slang/*
+            build.em/Release
```

Comment:
Do we need to include every file under this directory? Are there any files we want to exclude or this is just to pickup libslang.a?